seastar/src/sleep.cc

```diff
@@ -3,5 +3,13 @@
 namespace seastar_ffi {
 namespace sleep {
 
+VoidFuture steady_sleep(int64_t nanos) {
+    co_await seastar::sleep(std::chrono::nanoseconds(nanos));
+}
+
+VoidFuture lowres_sleep(int64_t nanos) {
+    co_await seastar::sleep<seastar::lowres_clock>(std::chrono::nanoseconds(nanos));
+}
```

Comment:
Is there a reason why you didn't add a manual clock sleep?

Reply:
For some unknown reason I though `seastar::sleep` supports only steady and manual clocks. Of course, manual clock sleep can be added and I did that.

There is a slight problem with testing `manual_sleep`. I would like to spawn a task using `seastar::spawn` that advances `ManualClock` in a loop and then call and await `manual_sleep`. Unfortunately, I can't await `seastar::spawn` here and without awaiting `seastar::spawn` currently doesn't have any effect. I can write a complete test after someone fixes that. That's why there is only a very simple smoke test of `manual_sleep` now.